include/read/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Создаем папку сборки для OpenCV
+file(MAKE_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build")
+# Вызываем команды для сборки OpenCV через execute_process
+execute_process(
+    COMMAND ${CMAKE_COMMAND} -S "${CMAKE_SOURCE_DIR}/3rdparty/opencv" -B "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build"
+    WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build"
+)
+execute_process(
+    COMMAND ${CMAKE_COMMAND} --build "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build"
+    WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/opencv/build"
+)
+set(INCLUDE_HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/reader_img.hpp")
+set(SRC_FILES "${CMAKE_SOURCE_DIR}/src/read/Reader_img.cpp")
+add_library(ReadLib STATIC ${INCLUDE_HEADERS} ${SRC_FILES})
+
+set_target_properties(ReadLib PROPERTIES LINKER_LANGUAGE CXX)
+
+target_include_directories(ReadLib PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
if include is already an include directory, remove this and write
```cpp
#include "read/reader_img.hpp"
```
in your cpp file

Reply:
same in test_read